extras/ams_wrapper/tests/unit/test_classification_attributes_model.py

```diff
@@ -72,28 +72,27 @@ def fake_output_config() -> Dict[str, ModelOutputConfiguration]:
 
 
 @pytest.mark.parametrize("inference_output,expected_response", [
-                        (MOCK_INFERENCE_OUTPUT,
-                         {"type": "classification", "subtype": None,
-                          "classifications": [{"attributes": [
-                              {"name": "color", "value": "yellow", "confidence": 0.3},
-                              {"name": "color", "value": "gray", "confidence": 0.2},
-                              {"name": "color", "value": "blue", "confidence": 0.2},
-                              {"name": "color", "value": "red", "confidence": 0.1},
-                              {"name": "color", "value": "green", "confidence": 0.1},
-                              {"name": "color", "value": "white",
-                                  "confidence": 0.05},
-                              {"name": "color", "value": "black", "confidence": 0.05}]},
-                              {"attributes": [
-                                  {"name": "type", "value": "van",
-                                      "confidence": 0.5},
-                                  {"name": "type", "value": "car",
-                                      "confidence": 0.2},
-                                  {"name": "type", "value": "bus",
-                                      "confidence": 0.2},
-                                  {"name": "type", "value": "truck",
-                                   "confidence": 0.1},
-                              ]}]}
-                         )])
+    (MOCK_INFERENCE_OUTPUT,
+     {
+         "inferences": [
+             {
+                 "type": "classification",
+                 "subtype": "type",
+                 "classification": {
+                     "tag": "van"
```

Comment:
I don't see confidence value anywhere in those expected responses. Is it not required?